src/rgw/rgw_admin_opt_user.h

```diff
@@ -0,0 +1,15 @@
+#ifndef CEPH_RGW_ADMIN_OPT_USER_H
+#define CEPH_RGW_ADMIN_OPT_USER_H
+
+#include "rgw_rados.h"
+#include "rgw_user.h"
+
+// This header and the corresponding source file contain handling of the following commads / groups of commands:
+// User
+
+int handle_opt_user_create(const std::string& subuser, RGWUserAdminOpState& user_op, RGWUser& user);
```

Comment:
missing user commands: modify, info, rm, suspend ...

Reply:
I haven't added methods for those because initially there was no space/readability gain because of that -- most of those functions are one-liners. I will add them once I add the appropriate class, thanks!